release-schema.json

```diff
@@ -19,7 +19,8 @@
           "deprecated": {
             "description": "This field is deprecated in favor of `relatedBids`, to allow an award to combine multiple bids.",
             "deprecatedVersion": "1.2"
-          }
+          },
+          "minLength": 1
```

Comment:
We can't run schema-strict, because bids is a recommended extension that is versioned with OCDS, and this would make data from 1.1.5 invalid in 1.2.0. Breaking changes are only allowed in major versions (2.0.0).

Reply:
ah! okay I'll remove all the validation bits